spec/rack/dev-mark/middleware_spec.rb

```diff
@@ -91,4 +92,14 @@
       expect(body).to eq(["response"])
     end
   end
+  context "redirection" do
+    let(:status) { 302 }
+    let(:body) { ['<html><head></head><body></body></html>'] }
+    it "does not insert dev mark" do
+      status, headers, body = subject.call({})
+      expect(status).to eq(302)
+      expect(headers).to include('Content-Type' => 'text/html; charset=utf-8')
+      expect(body).to eq(['<html><head></head><body></body></html>'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.